src/libraries/Microsoft.PowerFx.Connectors/Tabular/ServiceCapabilitiesExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    internal static class ServiceCapabilitiesExtensions
```

Comment:
>internal static class ServiceCapabilitiesExtensions [](http://example.com/codeflow?start=4&length=51)

Will this methods be useful for SNow and DV? if so, I wouldn't mind having them public and can we add some xml comments on them?

Reply:
yeah, we need to make them public as part of InternalsVisibleTo removal